internal/test/testdata/engine/case_29.yaml

```diff
@@ -0,0 +1,138 @@
+# yaml-language-server: $schema=../.jsonschema/EngineTestCase.schema.json
+---
+description: "Resource policy with REQUIRE_PARENTAL_CONSENT_FOR_ALLOWS"
+inputs:
+  [
+    {
+      "requestId": "test",
+      "actions": [
+        "create", # exists, falls through, allow
+        "delete", # exists, falls through, no match
+        "nonaction", # doesn't exist, falls through, no match (even with matching rule in rule table)
+        "acme_action" # doesn't exist, falls through, match (implicit DENY)
```

Comment:
This case is the same as `nonaction` above (no match in target scope with eventual `ALLOW` in parent), therefore can probably be removed?